config/environments/development.rb

```diff
@@ -10,10 +10,23 @@
   config.eager_load = false
 
   # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
+  config.consider_all_requests_local = true
   config.action_controller.perform_caching = false
 
   # Don't care if the mailer can't send.
+  config.action_mailer.delivery_method = :smtp
+
+  config.action_mailer.smtp_settings = {
+      address: "smtp.mandrillapp.com",
+      port: 587,
+      domain: "example.com",
+      authentication: "plain",
+      user_name: ENV["MANDRIL_USERNAME"],
+      password: ENV["MANDRIL_PASSWORD"],
+      enable_starttls_auto: true
+  }
+  config.action_mailer.default_url_options = { host: ENV['DEFAULT_URL_OPTION'] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.